
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.0.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/home/apithawa/ATM-TCR/main.py", line 8, in <module>
    import torch
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/__init__.py", line 643, in <module>
    from .functional import *  # noqa: F403
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/functional.py", line 6, in <module>
    import torch.nn.functional as F
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/nn/__init__.py", line 1, in <module>
    from .modules import *  # noqa: F403
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/nn/modules/__init__.py", line 2, in <module>
    from .linear import Identity, Linear, Bilinear, LazyLinear
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/nn/modules/linear.py", line 6, in <module>
    from .. import functional as F
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/nn/functional.py", line 11, in <module>
    from .._jit_internal import boolean_dispatch, _overload, BroadcastingList1, BroadcastingList2, BroadcastingList3
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/_jit_internal.py", line 28, in <module>
    import torch.package._mangling as package_mangling
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/package/__init__.py", line 12, in <module>
    from .package_importer import PackageImporter
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/package/package_importer.py", line 16, in <module>
    from ._directory_reader import DirectoryReader
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/package/_directory_reader.py", line 17, in <module>
    _dtype_to_storage = {data_type(0).dtype: data_type for data_type in _storages}
  File "/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/package/_directory_reader.py", line 17, in <dictcomp>
    _dtype_to_storage = {data_type(0).dtype: data_type for data_type in _storages}
/home/apithawa/.conda/envs/torch_env_test/lib/python3.9/site-packages/torch/package/_directory_reader.py:17: UserWarning: Failed to initialize NumPy: _ARRAY_API not found (Triggered internally at  /opt/conda/conda-bld/pytorch_1634272204863/work/torch/csrc/utils/tensor_numpy.cpp:68.)
  _dtype_to_storage = {data_type(0).dtype: data_type for data_type in _storages}
The `device` argument should be set by using `torch.device` or passing a string as an argument. This behavior will be deprecated soon and currently defaults to cpu.
The `device` argument should be set by using `torch.device` or passing a string as an argument. This behavior will be deprecated soon and currently defaults to cpu.
The `device` argument should be set by using `torch.device` or passing a string as an argument. This behavior will be deprecated soon and currently defaults to cpu.
[INDEP] {} ----------------
6118 17958 1204 22762
accuracy   0.6011
auc        0.7905
f1macro    0.5467
f1micro    0.6011
loss       345088.5500
precision0 0.8356
precision1 0.5590
recall0    0.2541
recall1    0.9498

